packages/api-client/src/v2/features/app/App.vue

```diff
@@ -106,9 +102,8 @@ const { store, workspaces, activeWorkspace, setWorkspaceId, createWorkspace } =
 useColorMode({ workspaceStore: store })
 
 /** Sidebar state and selection handling. */
-const sidebarState = useSidebarState({
+const sidebarState = useAppSidebar({
```

Comment:
Lets please call this initializeAppSidebar and put it into the helpers folder.

This prevents people from trying to consume this outside of the top level.